app/controllers/application_controller.rb

```diff
@@ -5,6 +5,18 @@ class ApplicationController < ActionController::Base
 
   before_filter :configure_permitted_parameters, if: :devise_controller?
 
+  def after_sign_in_path_for(resources)
```

Comment:
Unused method argument - `resources`. If it's necessary, use `_` or `_resources` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.